### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by LuckyDucko** [LuckyDucko/Mopups#10] [Announcements]
Mopups is up and running! (at least on my minimal testing it is!)

Would love to hear feedback from anyone having any issues with it, there are some definite things that needs to be sorted surrounding this repo, and also some nice to haves regarding the codebase itself that im interested in doing.

<hr /><em>This discussion was created from the release <a href='https://github.com/LuckyDucko/Mopups/releases/tag/'>1.0.0</a>.</em>

**Comment by kumaresan-cgvak** [comment]
@LuckyDucko Please update the Readme file accordingly.

**Reply by LuckyDucko** [reply]
All sorted, I'll give it a proper revise in the near future
